plotting/plottingUtils/inputManager.h

```diff
@@ -4,17 +4,21 @@
 #include "manager/MaCh3Logger.h"
 #include "manager/YamlHelper.h"
 #include "manager/MaCh3Exception.h"
+#include "mcmc/MCMCProcessor.h"
 
 // Other plotting includes
 #include "plottingUtils.h"
 
 namespace MaCh3Plotting {
 
+// EM: will move this somewhere more sensible
```

Comment:
We have bad double
https://github.com/mach3-software/MaCh3/blob/63d79bfe33e372534347fd7be456744584bc112d/samplePDF/Structs.h#L38

maybe we should use it?

Reply:
yeah good shout, probably some of the other parts of structs.h might be useful for plotting in future. e.g. making debug plots of splines etc.